wsmaster/che-core-api-devfile/src/test/java/org/eclipse/che/api/devfile/server/convert/DefaultEditorProvisionerTest.java

```diff
@@ -31,15 +31,19 @@
  */
 public class DefaultEditorProvisionerTest {
 
-  private static final String DEFAULT_EDITOR_ID = "org.eclipse.che.theia";
+  private static final String DEFAULT_EDITOR_NAME = "theia";
   private static final String DEFAULT_EDITOR_VERSION = "1.0.0";
-  private static final String DEFAULT_EDITOR_REF = DEFAULT_EDITOR_ID + ":" + DEFAULT_EDITOR_VERSION;
+  private static final String DEFAULT_EDITOR_PUBLISHER = "eclipse";
+  private static final String DEFAULT_EDITOR_REF =
+      DEFAULT_EDITOR_PUBLISHER + "/" + DEFAULT_EDITOR_NAME + "/" + DEFAULT_EDITOR_VERSION;
 
-  private static final String DEFAULT_TERMINAL_PLUGIN_ID = "org.eclipse.che.theia-terminal";
-  private static final String DEFAULT_TERMINAL_PLUGIN_REF = DEFAULT_TERMINAL_PLUGIN_ID + ":0.0.4";
+  private static final String DEFAULT_TERMINAL_PLUGIN_NAME = "theia-terminal";
+  private static final String DEFAULT_TERMINAL_PLUGIN_REF =
+      DEFAULT_EDITOR_PUBLISHER + "/" + DEFAULT_TERMINAL_PLUGIN_NAME + "/0.0.4";
 
-  private static final String DEFAULT_COMMAND_PLUGIN_ID = "org.eclipse.che.theia-command";
-  private static final String DEFAULT_COMMAND_PLUGIN_REF = DEFAULT_COMMAND_PLUGIN_ID + ":v1.0.0";
+  private static final String DEFAULT_COMMAND_PLUGIN_NAME = "theia-command";
+  private static final String DEFAULT_COMMAND_PLUGIN_REF =
+      DEFAULT_EDITOR_PUBLISHER + "/" + DEFAULT_COMMAND_PLUGIN_NAME + "/v1.0.0";
```

Comment:
We could drop `DEFAULT` from these vars for readability.

Reply:
changed